### ISSUE TITLE
Is it possible to draw fretboard diagram with guitar tabs?

### ISSUE THREAD

**Issue by DocRace** [opensheetmusicdisplay/opensheetmusicdisplay#1010]
I've got some guitar tabs in .xml. It rendered well in Musesocre with fretboard diagram alongside the chord notation, while in OSMD, I couldn't find an option to show them.
<img width="1186" alt="image" src="https://user-images.githubusercontent.com/13401166/114814187-3e686500-9de6-11eb-8d7d-a1b0f3dd533f.png">

**Comment by sschmidTU** [comment]
No, our renderer Vexflow can't do this. We would have to write or integrate a complete fretboard diagram library.

**Comment by abalter** [comment]
Hey, the library already exists, and is by the same author as Vexflow, [vexchords](https://github.com/0xfe/vexchords). So it should be relatively easy to integrate.

Actually, I wish that vexchord, vexflow, and vextab were already integrated so they could be combined.

https://github.com/0xfe/vexchords

**Comment by sschmidTU** [comment]
Oh, i've even starred it, but forgot about it, thanks for the hint!
This would of course be nice to integrate into OSMD, though it would also be quite a lot of work, and we don't have that many tab users. This would be great for contribution.

**Comment by abalter** [comment]
This is an aside, but does OSMD render guitar tab? It is part of the musicxml spec.

**Comment by sschmidTU** [comment]
@abalter yes, try the OSMD Function Test - Tabulature sample in the [demo](https://opensheetmusicdisplay.github.io/demo/).

**Comment by abalter** [comment]
Cool! Is it possible to have staff notation with guitar tab accompaniment? In other words, the notation is not the notes of the guitar  tab. They are independent, but rendered together.

**Comment by sschmidTU** [comment]
Yes.

**Comment by abalter** [comment]
Where could I find an example?

**Comment by abalter** [comment]
Ok. Maybe I can figure it out myself.

**Comment by Rakkkking** [comment]
nice!  i need guitart chord , Instead of Cm Dm
